config/config.go

```diff
@@ -129,11 +147,15 @@ type Gen2Config struct {
 type VPCProviderConfig struct {
 	Enabled              bool   `toml:"vpc_enabled" envconfig:"VPC_ENABLED"`
 	VPCBlockProviderName string `toml:"vpc_block_provider_name" envconfig:"VPC_BLOCK_PROVIDER_NAME"`
-	EndpointURL          string `toml:"vpc_endpoint_url" envconfig:"VPC_ENDPOINT_URL"`
+	EndpointURL          string `toml:"gc_riaas_endpoint_url"`
+	TokenExchangeURL     string `toml:"gc_token_exchange_endpoint_url"`
+	APIKey               string `toml:"gc_api_key" json:"-"`
```

Comment:
We can use bluemix API KEY, do we need it again?

Reply:
Yes gc has separate API key